Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
```

Comment:
We should use CEKit instead of direct Dockerfile, as done in https://github.com/kiegroup/kogito-operator/pull/784

Reply:
done